win32ss/user/ntuser/menu.c

```diff
@@ -3023,7 +3023,14 @@ static BOOL FASTCALL MENU_ShowPopup(PWND pwndOwner, PMENU menu, UINT id, UINT fl
         if ((y - height) < monitor->rcMonitor.top || y >= monitor->rcMonitor.bottom)
             y = monitor->rcMonitor.bottom - height;
         else
-            y -= height;
+        {
+            AdjHgt = y + UserGetSystemMetrics(SM_CYMENUSIZE) +
+                     2 * UserGetSystemMetrics(SM_CYDLGFRAME);
+            if (AdjHgt >= monitor->rcMonitor.bottom)
+                y -= height;
+            else
+                y = AdjHgt - height;
```

Comment:
instances of variables should begin not with capital letters 4/4
```suggestion
                y = adjHgt - height;
```